.github/workflows/update-version.yml

```diff
@@ -1,6 +1,7 @@
 name: Update VM version
 
 on:
+  workflow_dispatch:
```

Comment:
Please comment why we need the "workflow_dispatch" in the files.

Reply:
done